Makefile

```
@@ -124,12 +127,20 @@ minimal:				 #: Minimal build
 	$(MAKE) cmake BUILD_DIR=release BUILD_TYPE=release EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DVELOX_BUILD_MINIMAL=ON"
 	$(MAKE) build BUILD_DIR=release
 
-gpu:						 #: Build with GPU support
-	$(MAKE) cmake BUILD_DIR=release BUILD_TYPE=release EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DVELOX_ENABLE_GPU=ON -DVELOX_ENABLE_CUDF=ON"
+wave:			   	         #: Build with Wave GPU support
+	$(MAKE) cmake-wave BUILD_DIR=release BUILD_TYPE=release
+	$(MAKE) build BUILD_DIR=release
+
+cudf:			   	         #: Build with CUDF GPU support
```

Nit: in a sentence, cuDF is the preferred capitalization.
```suggestion
cudf:			   	         #: Build with cuDF GPU support
```

---

Fixed!